### DISCUSSION TITLE
Enable anonymous contributions?

### DISCUSSION THREAD

**Discussion by alexpdp7** [community/community#73316] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Is there a way to make an anonymous contribution?

**Comment by anjuan** [comment, marked as answer]
Hi @alexpdp7, when you [make a sponsorship,](https://docs.github.com/en/sponsors/sponsoring-open-source-contributors/sponsoring-an-open-source-contributor-through-github#sponsoring-an-account) you can determine who can see it. So, you can make a contributions, and you will not be seen as a sponsor for the account. Is this what you're trying to accomplish?

**Reply by alexpdp7** [reply]
OK, all the screens that I could access without providing payment information did not seem to indicate I would be able to remain anonymous. But after introducing my payment information I can choose this option.

I would make that more obvious, but this is doable.
